specs/latest/2.0/index.html

```diff
@@ -3386,6 +3492,19 @@ <h2 class="no-toc">Table of contents</h2>
         but that can be achieved by adjusting the data store offset accordingly.
     </div>
 
+    <h3><a name="DOM_UPLOAD_UNPACK_PARAMS">Pixel store parameters for <code>TexImageSource</code> uploads</a></h3>
+    <p>
+        <code>UNPACK_ALIGNMENT</code> and <code>UNPACK_ROW_LENGTH</code> are
+        ignored, as they determine row-stride-in-bytes, which is immutable for
+        TexImageSource objects.
+    </p>
+
+    <h3><a name="INVALID_BYTE_OFFSET"><code>byteOffset</code> and <code>ByteBuffer</code> args</a></h3>
+    <p>
+        If <code>byteOffset</code> is larger than the byte length of its
+        corresponding <code>ByteBuffer</code> arg, generate INVALID_VALUE.
+    </p>
```

Comment:
Despite the fact that it would cause duplication, this error should be defined in-place in all of the entry points that take this argument. Also, some of the entry points need to take a size argument (like bufferData and bufferSubData), and the semantic needs to be defined for that argument as well.